### ISSUE TITLE
Provide link to old file viewer for compare view

### ISSUE THREAD

**Issue by bobsilverberg** [mozilla/addons-code-manager#1472]
Depends on https://github.com/mozilla/addons-server/issues/13650

This is a spin-off from https://github.com/mozilla/addons-code-manager/issues/1471. Because we need new data from the API to implement the link to the compare page in the legacy file viewer, I am making this a separate issue.

This is about adding a link from the Code Manager to the legacy file viewer for a compare page. This link needs to be constructed using the file id of both the base and head versions, but we currently only have the file id of the head version.

#1471 will implement the link from a browse page in Code Manager to the legacy file viewer, and this issue can build on that after https://github.com/mozilla/addons-server/issues/13650 lands.

**Comment by diox** [comment]
Was blocked by https://github.com/mozilla/addons-server/issues/13650, which has since been implemented.

**Comment by ioanarusiczki** [comment]
@mirefly @bobsilverberg 

The link redirects correctly to the compare page from the old file viewer which opens in a new tab.

I randomly encountered time outs on the old file viewer page - this happens from admin tools too (See https://github.com/mozilla/addons-server/issues/13446) and in general, pages will load quite slow. 
Still, it is not reproducible as often as for the issue I filed on server.
I will check it on stage or further test some more on dev the next days.

Verified on dev with FF74(Win10).
